pages/_document.js

```diff
@@ -1,6 +1,7 @@
+import React from 'react'
 import Document, { Head, Main, NextScript } from 'next/document'
 import { ServerStyleSheet } from 'styled-components'
-import Router from 'next/router'
+// FIXME: import Router from 'next/router'
```

Comment:
This doesn't seem to be used in the file. Can I just delete the statement, or is there a special reason for this?